gradle.properties

```diff
@@ -0,0 +1,28 @@
+org.gradle.caching=true
+group=io.ballerina.stdlib
+version=0.1.0-SNAPSHOT
+#dependency versions
+ballerinaLangVersion=2201.2.0-20220607-123700-52fedf4d
+ballerinaTomlParserVersion=1.2.2
+checkstylePluginVersion=8.18
+commonsLang3Version=3.8.1
+slf4jVersion=1.7.30
+protoGoogleCommonsVersion=1.17.0
```

Comment:
Let's remove all the unnecessary dependencies and keep only the ones that are currently needed.